tests/extension/oneapi_device_global/device_global_common.h

```diff
@@ -90,6 +90,17 @@ struct value_helper {
    */
   static void change_val(T& value, const int new_val = 1) { value = new_val; }
 
+  /**
+   * @brief The function changes value from the first parameter to
+   * value from the second parameter of the same type
+   * Disabled if T is int to avoid function ambiguous
+   */
+  template <typename Ty = T>
+  static typename std::enable_if_t<!std::is_same_v<Ty, int>> change_val(
+      T& value, const T new_val) {
```

Comment:
```suggestion
      T& value, const T& new_val) {
```
or remove the top-level `const`, as this is not fundamental here.

Reply:
Added reference to `new_val`